File: java/0217.存在重复元素.java
class Solution {
    // 1. O(n) t:5ms(88%) O(n) m:49.5M(77%) 哈希表
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> set = new HashSet<Integer>();
        for (int num : nums) {
            if (!set.add(num)) return true;
        }
        return false;
    }
}

File: python/0217.存在重复元素.py
class Solution:
    # 1. O(n) t:92ms(17%) O(n) m:25.4M(29%) 哈希表
    def containsDuplicate(self, nums: List[int]) -> bool:
        return True if Counter(nums).most_common(1)[0][1] > 1 else False
    
    # ...